### PR TITLE
Clean up: specimen set

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions

MAJOR changes:
none

SPECIAL ATTENTION:
- removed `quantity` from concept schema, but property won't disappear; each specimen set gets a more specific property (`numberOfXY`) which askes for the same information

### CODE REVIEW

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@apdavison @Peyman-N could one of you confirm in addition (also to make sure that you are aware of this change)

File: schemas/research/specimenSet.schema.tpl.json [hunk 1]
```diff
@@ -21,21 +21,17 @@
...
-      "type": "integer",
-      "_instruction": "Enter the number of specimen that belong to this set."
+      "_instruction": "Enter a lookup label for this specimen set that may help you to find this instance more easily."
     },
     "species": {
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Add the species and/or the strain (a sub-type of a genetic variant of species) of all specimen in this set.",
+      "_instruction": "Add the species and/or strain (a sub-type of a genetic variant of species) of all specimen in this set.",
```
Comment (Peyman-N) [inline schemas/research/specimenSet.schema.tpl.json:34]: Just a suggestion maybe `a sub-type of a species with a unique genetic profile` is a little bit more specific. 
`Add the species and/or strain (a sub-type of a species with a unique genetic profile) of all specimen in this set.` 
I am not a biologist so I am not sure :sweat_smile:

**Review by Peyman-N — COMMENTED** [review]
Just another remark I think "specimen" should be pluralized as "specimens" since it refers to multiple specimens. 
`      "_instruction": "Add the biological sex of all specimens in this set.",`

File: schemas/research/specimenSet.schema.tpl.json (hunk 1 above)
Comment (aeidi89) [reply schemas/research/specimenSet.schema.tpl.json:34]: Having worked a bit with animals, for me the description of strain is not necessary. However, if it is helpful I would mind keeping it. Good to include all :)

Comment (lzehl) [reply schemas/research/specimenSet.schema.tpl.json:34]: I have the same opinion here. For me a species can be a strain or simple species definition. I would not need the additional information in brackets. @Peyman-N I'm not sure that your suggestion makes it more specific... do you insist on an update here?

Comment (Peyman-N) [reply schemas/research/specimenSet.schema.tpl.json:34]: I am okay with it, so i would merge it

**Review by Peyman-N — APPROVED** [review]

**Pull request merged**